bin/node/runtime/src/lib.rs

```diff
@@ -714,6 +715,12 @@ parameter_types! {
 	pub const MaxDepth: u32 = 32;
 	pub const StorageSizeOffset: u32 = 8;
 	pub const MaxValueSize: u32 = 16 * 1024;
+	pub DeletionWeightLimit: Weight = Perbill::from_percent(20) *
+		RuntimeBlockWeights::get().max_block;
+	pub DeletionQueueDepth: u32 = ((DeletionWeightLimit::get() / (
+			<Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(1) -
+			<Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(0)
+		)) / 5) as u32;
```

Comment:
This parameter is to limit the decoding costs of the queue. However, those costs are included in `DeletionWeightLimit`. It is just to prevent that the decoding costs become too larger or even larger than `DeletionWeightLimit`.